src/core/bittorrent/session.cpp

```diff
@@ -202,6 +202,12 @@ Session::Session(QObject *parent)
     configure();
     connect(pref, SIGNAL(changed()), SLOT(configure()));
 
+    // Network configuration monitor
+    connect(&network_manager, SIGNAL(onlineStateChanged(bool)), SLOT(onlineStateChanged(bool)));
+    connect(&network_manager, SIGNAL(  configurationAdded(const QNetworkConfiguration&)), SLOT(networkConfigurationChange(const QNetworkConfiguration&)));
```

Comment:
You probably did this to make it more beautiful, but there is no need.
Remove the space in SIGNAL()